<ui version="4.0" >
 <class>PrintWidget</class>
 <widget class="QWidget" name="PrintWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QRadioButton" name="rbPrint" >
         <property name="text" >
          <string>Print</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbView" >
         <property name="text" >
          <string>View as PDF</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QGroupBox" name="gbPrint" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Print settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbPrintCmd" >
            <property name="text" >
             <string>Printer command:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLineEdit" name="lePrintCmd" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbPrintCmd" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>325</width>
              <height>2</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbView" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Viewer settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QRadioButton" name="rbOpen" >
            <property name="text" >
             <string>Open in viewer application</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Command:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leOpenCmd" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="rbSave" >
            <property name="text" >
             <string>Save to disk</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowDialog" >
     <property name="text" >
      <string>Show this dialog before start printing</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>249</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>leOpenCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>249</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel" >
     <x>284</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPrintCmd</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePrintCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>206</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPrintCmd</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbPrintCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>327</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbPrint</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbPrint</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>62</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbView</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbView</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>62</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
